debian/source/format

```diff
@@ -1 +1 @@
-3.0 (native)
+3.0 (quilt)
```

Comment:
My attempt at building this locally with `sbuild` or `fakeroot dpkg-buildpackage` failed:
```
dpkg-source: error: can't build with source format '3.0 (quilt)': no upstream tarball found at ../cbmconvert_2.1.4.orig.tar.{bz2,gz,lzma,xz}
```
After I reverted the change to this file as well as removed the `-1` from `debian/changelog`, I was able to build a package in both ways.

I think that it would be good if those tools worked out of the box on the repository. Sure, if this were included in a GNU/Linux distribution, then those changes should be applied to the distribution’s source code package.

Reply:
I realized that plain `dpkg-buildpackage --no-sign` would work without `fakeroot`. The mention of `fakeroot` could be removed from `README.md` as part of this pull request, even though the line `Rules-Requires-Root: no` was not modified by this pull request.

Reply:
Yup, release tagging and release tarballs are missing from the repository. But you plan to provide those if I understand correctly.
Rules-Requires-Root are correct, package can be built as a normal user. No need to set SUID /SGID on any files, etc.

Reply:
Thank you. I successfully tested this by creating a source tarball.